org.adempiere.base/src/org/compiere/print/ReportEngine.java

```diff
@@ -356,14 +362,42 @@ public LayoutEngine getLayout()
 			layout();
 		return m_layout;
 	}	//	getLayout
+	
+	/**
+	 * 	Initialize Report Name
+	 */
+	public void initName()
+	{
+		
+		String processFileNamePattern = m_printFormat.getFileNamePattern();
+	 	if (m_info.getAD_Process_ID()>0) {
+			MProcess process = new MProcess(Env.getCtx(), m_info.getAD_Process_ID(), m_trxName);
+			if (process !=null && process.getFileNamePattern().length()>1) {
+				processFileNamePattern = process.getFileNamePattern();
+			}
+		}  
+		
+		if(Util.isEmpty(processFileNamePattern)) {
+	        
+	        Calendar cal = Calendar.getInstance();
+			SimpleDateFormat sdf = new SimpleDateFormat("yyyyMMddHHmmss");
+			String dt = sdf.format(cal.getTime());
+			
+			m_filename = m_printFormat.get_Translation("Name") + "_" + dt;
```

Comment:
this part change existing behavior when filenamepattern is empty. please document the changes in the ticket.